include/configs/ts7100.h

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (C) 2019-2022 Technologic Systems dba embeddedTS
+ *
+ * SPDX-License-Identifier:	GPL-2.0+
+ *
+ * Auto-included in files that include common.h
+ */
+#ifndef __TS7100_CONFIG_H
+#define __TS7100_CONFIG_H
+
+#include <asm/arch/imx-regs.h>
+#include <linux/sizes.h>
+#include "mx6_common.h"
+#include <asm/mach-imx/gpio.h>
+
+/* SPL options in include/configs */
+#include "imx6_spl.h"
+
+/* Env is at the 1MB boundary in emmc boot partition 0 */
+#define CONFIG_SYS_MMC_ENV_DEV  	0 /* mmcblk0 */
+#define CONFIG_SYS_MMC_ENV_PART 	1 /* boot0 */
+
+#define STATUS_LED_RED                  0
+#define STATUS_LED_GREEN                1
+#define STATUS_LED_BLUE                 2
+
+#define STATUS_LED_BIT                  STATUS_LED_RED
+#define STATUS_LED_STATE                STATUS_LED_ON
+#define STATUS_LED_PERIOD               (CONFIG_SYS_HZ / 2)
+
+/* Size of malloc() pool */
+#define CONFIG_SYS_MALLOC_LEN		(16 * SZ_1M)
+
+#define CONFIG_MXC_UART
+#define CONFIG_MXC_UART_BASE		UART1_BASE
+
+#define BOOT_TARGET_DEVICES(func) \
+	func(USB, usb, 0) \
+	func(MMC, mmc, 0) \
+	func(DHCP, dhcp, na)
+#define CONFIG_BOOTCOMMAND \
+	"run distro_bootcmd;"
+
+#define CONFIG_EXTRA_ENV_SETTINGS \
+	"fdt_high=0xffffffff\0" \
+	"initrd_addr=0x83800000\0" \
+	"initrd_high=0xffffffff\0" \
+	"autoload=no\0" \
+	"clearenv=env default -f -a && env save;\0" \
+	"emmc_dev=0\0" \
```

Comment:
```suggestion
```
Not used as far as I can see, should be removed.

Reply:
Useful for scripts that want to know whether eMMC is `mmcblk0` or `mmcblk1`.

Reply:
And, BTW, the TS-7250v3 has it too !-)

Reply:
Removed.